addons/guide_08_recommended_servers/manifest.json

```
@@ -0,0 +1,38 @@
+{
+  "api_version": "0.1",
+  "id": "guide_08_recommended_servers",
+  "name": "Guide: recommended servers",
+  "translatable": false,
+  "type": "guide",
+  "conditions": {
+    "enabled_features": ["recommendedServers"]
+  },
+  "guide": {
+    "id": "recommended_servers",
+    "title": "Why are certain locations recommended?",
+    "advanced": true,
+    "image": "qrc:/addons/guide_08_recommended_servers/image.svg",
+    "blocks": [
+      {
+          "id": "c_1",
+          "type": "text",
+          "content": "When you use Mozilla VPN, your internet activity is encrypted and routed through servers based in a location of your choice."
+      },
+      {
+          "id": "c_2",
+          "type": "text",
+          "content": "Sometimes, you might pick a location for a specific purpose; for example, if you're traveling internationally, you could choose a server location in your home country so that your banking apps function more smoothly."
```

```suggestion
          "content": "Sometimes, you might pick a location for a specific purpose; for example, if you’re traveling internationally, you could choose a server location in your home country so that your banking apps function more smoothly."
```

This should have tripped the linter, but it didn't (even before the add-on was marked as no-translate).
https://github.com/mozilla-mobile/mozilla-vpn-client/actions/workflows/translations.yaml?query=branch%3Arec-servers-guide

I need to check if we are looking into add-ons.

Please replace all `'` with `’`.